src/consensus/pbft_manager.cpp

```diff
@@ -843,7 +834,8 @@ void PbftManager::identifyBlock_() {
 
   if (round == 1 || (round >= 2 && giveUpNextVotedBlock_())) {
     // Identity leader
-    std::pair<blk_hash_t, bool> leader_block = identifyLeaderBlock_(votes_);
+    auto proposal_votes = vote_mgr_->getProposalVotes(round);
+    std::pair<blk_hash_t, bool> leader_block = identifyLeaderBlock_(proposal_votes);
```

Comment:
std::move(proposal_votes)

Reply:
Move vote_mgr_->getProposalVotes(round) in identifyLeaderBlock_ function